components/PrivKey/PrivKey.tsx

```diff
@@ -98,7 +97,7 @@ const PrivKey: React.FunctionComponent<PrivKeyProps> = ({ address, keyType, priv
               style={{
                 flexBasis: '100%',
                 textAlign: 'center',
-                fontFamily: fixedWidthFont,
+                fontFamily: 'verdana',
```

Comment:
I changed this screen for legacy propose... because, as you know, the App are not using this screen anymore. I don't know if in some future we are going to rescue some of this functionalities... who knows...